.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,9 +2,9 @@
 
 Description of your PR. Fixes # (issue) (if applicable)
 
-## New press checklist (if applicable)
-
-- [ ] I added `mypress_press.py` in the `presses` directory
-- [ ] I added `MyPress` in `__init__.py` 
-- [ ] I updated the `README.md` with a 1 liner about my new press in the Available presses section
-- [ ] I added my press in the `default_presses` list in `tests/default_presses.py`
+## Checklist
```

Comment:
We could add:

- Ensure tests are working (`make test`)
- Code is formatted correctly (`make style`, on errors try fix with `make format`)